### PR TITLE
Support NewRelic v2 + Tests

### PR DESCRIPTION
**Pull request by mdemri**

### CODE REVIEW

**Comment by mdemri** [comment]
https://circleci.com/gh/stellarjs/stellarjs/173?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Yarn not found?

File: packages/mw-newrelic/test/mw-newrelic.test.js [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+import set from 'lodash/set';
+import middleware from '../src';
+
+describe('Test NR middleware use cases', () => {
+  it('Should pass request w/o queue to the next midddleware ', () => {
+    const req = {};
+    const val = 'success';
+    const next = () => Promise.resolve(val);
+
+    middleware(req, next)
```
Comment (sloops77) [inline packages/mw-newrelic/test/mw-newrelic.test.js]: you should add a mock for the newrelic lib to show that the calls happen as expected...

Comment (sloops77) [reply packages/mw-newrelic/test/mw-newrelic.test.js]: https://facebook.github.io/jest/docs/en/mock-functions.html

File: packages/mw-newrelic/test/mw-newrelic.test.js [hunk 2]
```diff
@@ -1,8 +1,37 @@
...
  */
-describe('no tests', () => {
-  it('no test', () => {
-    // fail();
+
+import Promise from 'bluebird';
+import set from 'lodash/set';
+import middleware from '../src';
+
+describe('Test NR middleware use cases', () => {
```
Comment (sloops77) [inline packages/mw-newrelic/test/mw-newrelic.test.js]: describe('New Relic middleware')

Comment (mdemri) [reply packages/mw-newrelic/test/mw-newrelic.test.js]: OK

File: packages/mw-newrelic/test/mw-newrelic.test.js (hunk 1 above)
Comment (mdemri) [reply packages/mw-newrelic/test/mw-newrelic.test.js]: New relic without config file acts like a mock (do nothing)
I'll check it anyway

**Pull request merged**